src/nim_status_client.nim

```diff
@@ -28,7 +31,14 @@ proc mainProc() =
     else:
       "/../fleets.json"
 
-  let status = statuslib.newStatusInstance(readFile(joinPath(getAppDir(), fleets)))
+  let
+    fleetConfig = readFile(joinPath(getAppDir(), fleets))
+    status = statuslib.newStatusInstance(fleetConfig)
+    mailserverController = mailserver_controller.newController(status)
+    mailserverWorker = mailserver_worker.newMailserverWorker(cast[ByteAddress](mailserverController.vptr))
+
+  # TODO: create and register an ipcWorker
```

Comment:
Curious about this comment, what would be the purpose of the ipcWorker?

Reply:
It would be a cleaner way to implement the IPC listener thread in https://github.com/status-im/status-desktop/pull/2043.

There's another aspect to that PR re: adding info to the Windows registry for deep links, and that will be properly handled by the installer program for Windows we have in the future, but we'll still need the IPC listener thread.

Reply:
> Do they "spam" a lot? 

The majority of the debug messages added here were already existing in the mailserver model. However, I'll swap over the newly added mailserver-specific ones to trace as suggested.